jbpm-audit/src/main/java/org/jbpm/process/audit/jms/AsyncAuditLogProducer.java

```diff
@@ -161,10 +159,11 @@ public void afterNodeTriggered(ProcessNodeTriggeredEvent event) {
     	}
     }
 
+
     @Override
     public void beforeNodeLeft(ProcessNodeLeftEvent event) {
-
-        
+        NodeInstanceLog log = (NodeInstanceLog) builder.buildEvent(event, null);
+        sendMessage(log, AFTER_NODE_LEFT_EVENT_TYPE, 1);
```

Comment:
It should be BEFORE_NODE_LEFT_EVENT_TYPE, shouldn't it?
So, now, it's logging it before leaving the node

Reply:
this can cause an incompatibility. So I wouldn't do that at least for now.